src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
 }
 
 #ifdef HAVE_SECCOMP
+static int syscall_filter_parse_one(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                ExecContext *c,
+                bool invert,
+                const char *t,
+                bool warn) {
+        int r;
+
+        if (*t == '@') {
+                const char * const *s;
+                static const char * const syscall_sets[] = {
+                        /* Default list */
+                        "@Default",
+                        "execve\0"
+                        "exit\0"
+                        "exit_group\0"
+                        "rt_sigreturn\0"
+                        "sigreturn\0",
```

Comment:
Hmm, I figure the biggest disccussion point will be to define what groups to define here, and what to call them...

None of the enum-like tokens we used to so far in unit files use CamelCase btw, they generally use lower-case strings, with dashes as word separators, I think we should follow this here too, i.e. call the groups "@default", "@network-io" and so on...